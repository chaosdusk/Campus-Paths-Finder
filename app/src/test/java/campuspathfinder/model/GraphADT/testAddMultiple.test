CreateGraph graph
#Add nodes
AddNode graph node1
AddNode graph node2
AddNode graph node3
AddNode graph node4
AddNode graph node5
#Add edges
AddEdge graph node1 node2 label
AddEdge graph node1 node3 abel
AddEdge graph node2 node4 bela
AddEdge graph node2 node5 abel
#Confirm
ListNodes graph
ListChildren graph node1
ListChildren graph node2