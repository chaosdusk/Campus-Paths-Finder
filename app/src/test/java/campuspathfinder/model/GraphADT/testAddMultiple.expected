created graph graph
#Add nodes
added node node1 to graph
added node node2 to graph
added node node3 to graph
added node node4 to graph
added node node5 to graph
#Add edges
added edge label from node1 to node2 in graph
added edge abel from node1 to node3 in graph
added edge bela from node2 to node4 in graph
added edge abel from node2 to node5 in graph
#Confirm
graph contains: node1 node2 node3 node4 node5
the children of node1 in graph are: node2(label) node3(abel)
the children of node2 in graph are: node4(bela) node5(abel)